crates/solidity/testing/snapshots/cst_output/ContractDefinition/member_using_directive/generated/0.4.11-success.yml

```diff
@@ -9,18 +9,18 @@ Errors: []
 
 Tree:
   - ContractDefinition: # 0..36 "contract Sample {\n  using x for *;\n}"
```

Comment:
I wonder if we should split adding names to CST snapshots to a different PR, to make sure that the parser changes here didn't change anything else.

Thoughts?

Reply:
I've made sure that it doesn't change name, see https://github.com/NomicFoundation/slang/pull/710/files/e11644685bf7d2e19c2d75ee399cb5aa51de61e4..17a4fce8d441851575680ab1ebf5ce703a38b14f (for some reason I can't start the PR diff from the base branch but from the first commit?) for the diff between the first commit here and the last.

Reply:
This is big enough of a change that I'd rather link the PR that causes changes with the output change itself and we can verify as per above that the naming changes didn't change the shape on its own, so I'd like to merge this as-is, if you don't mind :pray: